src/stacks-hierarchy/Root_ParkingViolationsReporting/Root_ScooterContactOperatorScreen.tsx

```diff
@@ -0,0 +1,285 @@
+import {isValidPhoneNumber} from 'libphonenumber-js';
+import {MessageInfoBox} from '@atb/components/message-info-box';
+import {useSendSupportRequestMutation} from '@atb/mobility/queries/use-send-support-request-mutation';
+import {useActiveShmoBookingQuery} from '@atb/mobility/queries/use-active-shmo-booking-query';
+import {getCurrentCoordinatesGlobal} from '@atb/GeolocationContext';
+import {useProfileQuery} from '@atb/queries';
+import {isValidEmail} from '@atb/utils/validation';
+import {FullScreenHeader} from '@atb/components/screen-header';
+import {getThemeColor} from './components/ScreenContainer';
+
+const supportItems: SupportType[] = [
+  SupportType.UNABLE_TO_OPEN,
+  SupportType.UNABLE_TO_CLOSE,
+  SupportType.REFUND,
+  SupportType.ACCIDENT_OR_BROKEN,
+  SupportType.OTHER,
+];
+
+export type Root_ScooterContactOperatorScreenProps =
+  RootStackScreenProps<'Root_ScooterContactOperatorScreen'>;
+
+export const Root_ScooterContactOperatorScreen = ({
+  route,
+}: Root_ScooterContactOperatorScreenProps) => {
+  const {vehicleId, operatorId} = route.params;
+  const style = useStyles();
+  const {theme} = useThemeContext();
+  const themeColor = getThemeColor(theme);
+  const {t} = useTranslation();
+  const {operatorName} = useVehicle(vehicleId);
+  const {phoneNumber: authPhoneNumberWithPrefix} = useAuthContext();
+  const {prefix: authPhonePrefix, phoneNumber: authPhoneNumber} =
+    getParsedPrefixAndPhoneNumber(authPhoneNumberWithPrefix);
+
+  const onSuccess = () => {
+    // TODO: navigate to next screen
+  };
+
+  const {
+    mutate: sendSupportRequest,
+    isLoading: isLoadingSupportRequest,
+    isError: isErrorSupportRequest,
+  } = useSendSupportRequestMutation(operatorId, onSuccess);
+
+  const {data: activeShmoBooking} = useActiveShmoBookingQuery();
+  const {data: customerProfile} = useProfileQuery();
+
+  const [selectedSupportType, setSelectedSupportType] = useState<SupportType>(
+    SupportType.OTHER,
+  );
+  const [isContactInfoPresent, setIsContactInfoPresent] =
+    useState<boolean>(true);
+  const [isPhoneNumberValid, setIsPhoneNumberValid] = useState<boolean>(true);
+  const [phoneNumber, setPhoneNumber] = useState(authPhoneNumber);
+  const [phonePrefix, setPhonePrefix] = useState(authPhonePrefix ?? '47');
+  const [email, setEmail] = useState<string>(customerProfile?.email ?? '');
+  const [isEmailValid, setIsEmailValid] = useState(true);
+  const [comment, setComment] = useState<string>('');
+  const [isCommentValid, setIsCommentValid] = useState(true);
+
+  const setAllInputIsValid = () => {
+    setIsCommentValid(true);
+    setIsContactInfoPresent(true);
+    setIsPhoneNumberValid(true);
+    setIsEmailValid(true);
+  };
+
+  const onSubmit = async () => {
+    const newIsCommentValid =
+      SendSupportRequestBodySchema.shape.comment.safeParse(comment).success;
+    const newPhoneNumberWithPrefix = '+' + phonePrefix + phoneNumber;
+    const newIsPhoneNumberValid =
```

Comment:
Maybe there is a different logic but this might work as well: `phoneNumber && isValidPhoneNumber(newPhoneNumberWithPrefix)`